python/lsst/pipe/base/execution_reports.py

```diff
@@ -218,11 +223,17 @@ def to_summary_dict(self, butler: Butler, do_store_logs: bool = True) -> dict[st
             - n_quanta_blocked: The number of quanta which failed due to
               upstream failures.
             - n_succeded: The number of quanta which succeeded.
+            And possibly, if human-readable is passed:
```

Comment:
I think the doc error is from this line. You might need to add a blank line above and below.